src/Config.h

```diff
@@ -71,6 +71,15 @@ COPYRIGHT (c) 2017 Mike Dunston
 // PROGRAMMING TRACK DCC SIGNAL PIN
 #define DCC_SIGNAL_PIN_PROGRAMMING 18
 
+/////////////////////////////////////////////////////////////////////////////////////
+//
+// DEFINE THE CURRENT SENSE ATTENUATION.  Default is 11, you may select 6 or 0
+// depending on your setup
+//
+#define ADC_ATTEN_DB_NUM ADC_ATTEN_DB_11
+//#define ADC_ATTEN_DB_NUM ADC_ATTEN_DB_6
```

Comment:
we can add ADC_ATTEN_DB_2_5 as well since it is a supported value.

Also for most cases where multiple values are supported instead of multiple defines I have been listing the possible values in the comments. Not sure which is better though.